dxr/plugins/clang/dxr-index.cpp

```diff
@@ -461,9 +461,12 @@ class IndexConsumer : public ASTConsumer,
     if (d->isThisDeclarationADefinition() ||
         d->isPure())  // until we have better support for pure-virtual functions
     {
-      beginRecord("function", d->getLocation());
-      recordValue("name", d->getNameAsString());
-      recordValue("qualname", getQualifiedName(*d));
+      SourceLocation functionLocation = d->getLocation();
```

Comment:
Do these add anything, like clarity, in your opinion, or are they just debugging expansions that didn't get removed?